client/src/plugins/camunda-plugin/start-instance-tool/util/__tests__/isExecutableSpec.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.
+ *
+ * Camunda licenses this file to you under the MIT; you may not use this file
+ * except in compliance with the MIT License.
+ */
+
+import isExecutable from '../isExecutable';
+
+import executableXML from './fixtures/executable.bpmn';
+import notExecutableXML from './fixtures/not_executable.bpmn';
+import collaborationExecutableXML from './fixtures/collaboration_executable.bpmn';
+import collaborationNotExecutableXML from './fixtures/collaboration_not_executable.bpmn';
+
+describe('plugins/start-instance/util - isExecutable', function() {
+
+  describe('single process', function() {
+
+    it('should find executable process', async function() {
+
+      // when
+      const executable = await isExecutable(executableXML);
+
+      // then
+      expect(executable).to.be.true;
+    });
+
+
+    it('should NOT find executable process', async function() {
```

Comment:
Today's episode of "No big deal but": Should we rename this test to "should find/detect non-executable process" instead?

Reply:
well,

What the test should do: verify no executable process is found
What the test should not do: verify a non-executable process is found

I know it's confusing, but do you know what I mean?

Reply:
👍